pubsub/tests/unit/pubsub_v1/subscriber/test_streaming_pull_manager.py

```diff
@@ -660,9 +660,9 @@ def test__on_response_with_leaser_overload():
         ]
     )
 
-    # Adjust message bookkeeping in leaser. Pick 99 messages, which is just below
+    # Adjust message bookkeeping in leaser. Pick 999 messages, which is just below
     # the default FlowControl.max_messages limit.
-    fake_leaser_add(leaser, init_msg_count=99, init_bytes=990)
+    fake_leaser_add(leaser, init_msg_count=999, init_bytes=990)
```

Comment:
The `fake_leaser_add()` method increase the total byte size for every added message by 10. For consistency, `init_bytes` should thus be adjusted to `9990`.

While probably out of this PR's scope, an even better solution would be to introduce a convenience parameter such as `assumed_message_size`, replacing the currently hardcoded value of 10. And possibly even replacing the `init_bytes` parameter, as the latter can then be computed automatically.

I have some spare time to open a PR for that.

Reply:
In short - the leaser keeps count of how many messages are being lease-managed, and their total size. This info is used by the streaming pull manager to compute the current load, and possibly pause/resume the message stream.

The leaser mock here mimics that bookkeeping, but currently adds a hardcoded value of 10 bytes to the tally for each added message, that's all. :)

Reply:
Got it, thank you!